sylius/resource-bundle/1.12/config/packages/sylius_resource.yaml

```diff
@@ -1,4 +1,4 @@
-# @see https://github.com/Sylius/SyliusResourceBundle/blob/master/docs/index.md
+# @see https://docs.sylius.com/sylius-stack/resource/index
```

Comment:
I'm still waiting for Magda answer for the right link between this one and https://stack.sylius.com/resource/index